### ISSUE TITLE
accessibility request

### ISSUE THREAD

**Issue by Shehryar157** [yasirkula/UnityChoiceOfGamesSaveManager#1]
As a visually impaired fan and near quite frequent CoG player, I can say it sucks when your character dyes at what I assume to roughly be midway through the game, the resulting reaction by at least me is not a pleasant one. I don't want to play a game for 3 days and some parts of night, then realize that I either pressed the wrong option, or that I simply dyed as the outcome of a choice I picked.
      Finding this program was a real relief, and I, along with my friend who happens to play CoG even more frequently than me were going to use this save manager, had it not been for the lack of accessibility.
      Put it simply, if I run the program and try to navigate around, my screen reader doesn't speak anything. So I technically can't really do anything. And even if I try to use OCR, I still can't do much. Could you perhaps edit the program's code so as to make it usable by blind folk as well? I, for one, would be appreciative if you were to do it or at least consider the suggestion or feedback, though I'm not sure if coining the text the latter term would be entirely accurate.

**Comment by yasirkula** [comment]
Thank you for creating this feature request! The game engine I'm using, Unity, doesn't have built-in support for screen readers. I've found a popular open-source extension which may work but would obviously take my time. I may give it a shot in the future but for the time being, unfortunately this feature won't be available. I'll keep this feature request open in the meantime.
